src/main/java/uk/co/ramp/covid/simulation/population/Adult.java

```diff
@@ -13,7 +13,8 @@
 
     Professions profession;
 
-    public Adult() {
+    public Adult(int age, Sex sex) {
+        super(age, sex);
```

Comment:
Should there be a check to ensure Adult age is between 18 and 65? Similarly for Infant, Child & Pensioner

Reply:
I've added in some checks for this. It's not particularly clear what to do in this situation but since It probably means the population is completely messed up so I'm throwing an exception!